src/main/java/seedu/address/logic/parser/profile/ProfileCommandParser.java

```diff
@@ -53,6 +54,12 @@ public ProfileCommand parse(String args) throws ParseException {
             return new AddProfileCommandParser().parse(args);
         case FindProfileCommand.COMMAND_OPTION:
             return new FindProfileCommandParser().parse(args);
+        case DeleteProfileCommand.COMMAND_OPTION:
+            return new DeleteProfileCommandParser().parse(args);
+        case EditProfileCommand.COMMAND_OPTION:
+            return new EditProfileCommandParser().parse(args);
+        case ViewProfilesCommand.COMMAND_OPTION:
+            return new ViewProfilesCommandParser().parse(args);
```

Comment:
Minor, but maybe we could put this in alphabetical order or group related commands together.